libs/ui-watt/src/lib/foundations/color/storybook/color.stories.ts

```diff
@@ -0,0 +1,38 @@
+import { moduleMetadata, Story, Meta } from '@storybook/angular';
+
+import { ColorComponent } from './color.component';
+import { ColorModule } from './color.module';
+
+export default {
+  title: 'Foundations/Color',
+  component: ColorComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [ColorModule],
+    }),
+  ],
+} as Meta<ColorComponent>;
+
+//👇 We create a “template” of how args map to rendering
+const Template: Story<ColorComponent> = (args) => ({
+  props: args,
+});
+
+//👇 Each story then reuses that template
+export const Color = Template.bind({});
+Color.parameters = {
+  docs: {
+    source: {
+      code: ` // Usage from SCSS / CSS (tip: hover over the color sample, and click to copy to clipboard):
+.my-element {
+  background: var(COLOR NAME);
+}
+
+// Usage from TypeScript:
+1. import { Colors, ColorHelperService } from '@energinet/watt';      
+2. Inject the ColorHelperService
+3. Use ColorHelperService.getColor(Colors.COLOR NAME);
```

Comment:
I can see now that GitHub "ate" part of my previous comment related to this line. The last line should've been: 

Suggestion: Consider changing "COLOR NAME" string to `<color-name>` to make it more explicit.

E.g.

```
 3. Use ColorHelperService.getColor(Colors.<color-name>);
```

Reply:
Roger that